packages/ffe-core/less/typography.less

```diff
@@ -186,6 +194,11 @@
     font-size: var(--ffe-fontsize-micro-text);
 }
 
+.ffe-strong-micro-text {
+    font-family: var(--ffe-g-font-strong);
+    font-size: var(--ffe-fontsize-micro-text);
+}
+
 .ffe-h1,
 .ffe-h2,
 .ffe-h3,
```

Comment:
Tror den burde ligge her ogås muligens (annen line-heigth på større skjermer). Sikkert bare og søke p å .ffe-micro-text i typographi.less og legge in samme steder
![image](https://github.com/SpareBank1/designsystem/assets/2248579/0b8636e1-4701-4061-b6c3-819f95032ce5)